neuralop/losses/data_losses.py

```diff
@@ -286,6 +286,77 @@ def __call__(self, y_pred, y, h=None, **kwargs):
         return self.rel(y_pred, y, h=h)
 
 
+class DissipativeLoss(object):
+    """Dissipative loss is used to avoid model blow-up and to encourage the model
+    to learn dissipative dynamics (particularly in chaotic systems). The dissipative
+    loss samples points uniformly and randomly in a spherical shell around the
+    attractor of a chaotic system (at user-specified radii) and encourages
+    that a learned model's predictions on this shell scale inwards by some
+    user-specified dissipative rule.
```

Comment:
Can you have some higher, less technical level intuition here? Readers can refer to the paper for details.